index.md

```diff
@@ -374,6 +376,30 @@ describe('Array', function() {
 });
 ```
 
+## Retry Tests
+
+You can choose to retry failed tests up to a certain number of times. This feature is designed to handle end-to-end tests (functional tests/Selenium...) where resources cannot be easily mocked/stubbed. **It's not recommended to use this feature for unit tests**.
+
+This feature does re-run `beforeEach/afterEach` hooks but not `before/after` hooks.
+
+```js
+describe('retries', function() {
+  // Retry all tests in this suite up to 4 times
+  this.retries(4);
+  
+  beforeEach(function () {
+    browser.get('http://www.yahoo.com');
```

Comment:
ah, another Selenium sauce where it [rigged test hooks](https://github.com/SeleniumHQ/selenium/blob/c10e8a955883f004452cdde18096d70738397788/javascript/node/selenium-webdriver/testing/index.js) to execute its own `Promise` chain always w/o return